skyportal/handlers/api/gcn.py

```diff
@@ -1954,3 +1957,58 @@ class Validator(Schema):
         except Exception as e:
             return self.error(f"Error generating summary: {e}")
         return self.success(data=contents)
+
+
+class LocalizationDownloadHandler(BaseHandler):
+    @auth_or_token
+    async def get(self, dateobs, localization_name):
+        """
+        ---
+        description: Download a GCN localization skymap
+        tags:
+          - localizations
+        parameters:
+          - in: path
+            name: dateobs
+            required: true
+            schema:
+              type: dateobs
+          - in: path
+            name: localization_name
+            required: true
+            schema:
+              type: localization_name
+        responses:
+          200:
+            content:
+              application/json:
+                schema: LocalizationHandlerGet
+          400:
+            content:
+              application/json:
+                schema: Error
+        """
+
+        with self.Session() as session:
+            localization = session.scalars(
+                Localization.select(session.user_or_token).where(
+                    Localization.dateobs == dateobs,
+                    Localization.localization_name == localization_name,
+                )
+            ).first()
+            if localization is None:
+                return self.error("Localization not found", status=404)
+
+            output_format = 'pdf'
```

Comment:
this is an error I guess, it downloaded the skymap as a pdf because of it.

Reply:
ooops.